<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagerSenderClass</class>
 <widget class="QMainWindow" name="PagerSenderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HackRF POCSAG Sender</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>POCSAG Message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLCDNumber" name="charCount">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="messageText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="msgCharset">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bitrate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Function:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="msgBitrate">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pager RIC:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Charset:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="messageType">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Message type:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="msgFunc">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pagerRIC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label1_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Character count:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="dateTime">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1_12">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Date/Time add:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>185</y>
      <width>171</width>
      <height>96</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>TX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="freq">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label1_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_9">
     <property name="geometry">
      <rect>
       <x>84</x>
       <y>32</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bandwidth">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bandwidth:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="useAmp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RF Gain:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="rfGain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>v1.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PagerSender.qrc"/>
 </resources>
 <connections/>
</ui>
